docs/AboutUs.adoc

```diff
@@ -4,53 +4,71 @@
 :imagesDir: images
 :stylesDir: stylesheets
 
+TODO: Update +
 AddressBook - Level 3 was developed by the https://se-edu.github.io/docs/Team.html[se-edu] team. +
```

Comment:
Should we change this part to our product name and change the link to our repo link?